### ISSUE TITLE
Info message 'Not loaded properly in Lovelace' for a card that is working fine

### ISSUE THREAD

**Issue by rpitera** [hacs/integration#1326]
## Installation details

| Description                | Value |
| -------------------------- | ----- |
| HACS version               | 1.1.0
| Home Assistant version     | 0.110.7
| Installation method for HA | HomeAssistant (formerly HASS.io) on Ubuntu VM

## Checklist

- [X] I'm running the newest version of HACS https://github.com/hacs/integration/releases/latest
- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read https://hacs.xyz/docs/issues
- [X] This issue is related to the backend of HACS.
- [X] This issue only contain 1 issue (if you have multiple issues, open one issue for each issue).

## Describe the issue

Getting an informational message on the main HACS page that:

```
Not loaded in Lovelace
You have 1 Lovelace elements that are not loaded properly in Lovelace.
```

When I track down the card in the frontend section, it shows that Mini Graph Card has the "Not Loaded" red ring around it.  Thing is, the Mini Graph Card is only used in two views and both of them are working fine.  They load properly and the data shows up in the cards.  I looked in the developer tools of Chrome to see if anything shows as an error in the console logs and nothing related to Mini Graph Card is there.  Nothing in the HA logs indicates an issue either.  

Every instance of the Mini Graph Card is working but HACS thinks something is wrong.  I'm not getting this message for any other HACS installed code except this card.

### Steps to reproduce

Not really applicable, but I did try removing and reinstalling the Mini Graph Card, only to see the same message in HACS after a reload and a reboot.

## Debug logs

<details>
  <summary>Logs</summary>

```2020-06-10 14:01:06 DEBUG (MainThread) [hacs] Configuration type: flow
2020-06-10 14:01:06 INFO (MainThread) [hacs] 
-------------------------------------------------------------------
HACS (Home Assistant Community Store)

Version: 1.1.0
This is a custom integration
If you have any issues with this you need to open an issue here:
https://github.com/hacs/integration/issues
-------------------------------------------------------------------

2020-06-10 14:01:07 DEBUG (MainThread) [hacs] Can update 468 repositories
2020-06-10 14:01:07 DEBUG (MainThread) [hacs.repository.integration.hacs.integration] Checking repository.
2020-06-10 14:01:08 DEBUG (MainThread) [hacs.repository.integration.hacs.integration] Running checks against 1.1.0
2020-06-10 14:01:09 INFO (MainThread) [hacs.repository.integration.hacs.integration] Registration complete
2020-06-10 14:01:10 INFO (MainThread) [hacs.data] Restore started
2020-06-10 14:01:11 INFO (MainThread) [hacs.data] Restore done
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] appdaemon_path: appdaemon/apps/
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] appdaemon: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] netdaemon_path: netdaemon/apps/
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] netdaemon: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] config_type: flow
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] debug: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] dev: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] frontend_mode: Grid
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] frontend_compact: True
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] frontend_repo: 
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] frontend_repo_url: 
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] onboarding_done: True
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] plugin_path: www/community/
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] python_script_path: python_scripts/
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] python_script: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] sidepanel_icon: mdi:store
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] sidepanel_title: HACS
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] theme_path: themes/
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] theme: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] country: ALL
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] experimental: False
2020-06-10 14:01:11 DEBUG (MainThread) [hacs.configuration] release_limit: 5
2020-06-10 14:02:19 INFO (MainThread) [hacs] Loading known repositories
2020-06-10 14:02:22 DEBUG (MainThread) [hacs] Starting recuring background task for installed repositories
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.data] Saving data
2020-06-10 14:02:23 DEBUG (MainThread) [hacs] Recuring background task for installed repositories done
2020-06-10 14:02:23 DEBUG (MainThread) [queueman.manager] Checking out tasks to execute
2020-06-10 14:02:23 DEBUG (MainThread) [queueman.manager] Starting queue execution for 37 tasks
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.integration.hacs.integration] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.integration.custom-components.alexa_media_player] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.integration.custom-components.weatheralerts] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.ofekashery.vertical-stack-in-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.canvas-gauge-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-slider-entity-row] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.button-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.upcoming-media-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-fold-entity-row] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.kalkih.mini-graph-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.integration.custom-components.sensor.plex_recently_added] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.flex-table-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-layout-card] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-state-switch] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-card-tools] Getting repository information
2020-06-10 14:02:23 DEBUG (MainThread) [hacs.repository.integration.hacs.integration] Running checks against 1.1.0
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.ofekashery.vertical-stack-in-card] Running checks against v0.3.3
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.integration.custom-components.weatheralerts] Running checks against 0.1.1
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-slider-entity-row] Running checks against 15
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-state-switch] Running checks against 8
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.canvas-gauge-card] Running checks against 0.4
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-layout-card] Running checks against 13
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-card-tools] Running checks against 10
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.integration.custom-components.sensor.plex_recently_added] Running checks against 0.3.2
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.upcoming-media-card] Running checks against 0.4.3
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-fold-entity-row] Running checks against 16
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.kalkih.mini-graph-card] Running checks against v0.9.3
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.flex-table-card] Running checks against master
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.integration.custom-components.alexa_media_player] Running checks against v2.8.4
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.button-card] Running checks against 3.3.6
2020-06-10 14:02:24 DEBUG (MainThread) [hacs.template] 'is_state' is undefined
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-card-preloader/lovelace-card-preloader.js from /config/www/community/lovelace-card-preloader/lovelace-card-preloader.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving calendar-card/calendar-card.js from /config/www/community/calendar-card/calendar-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving list-card/list-card.js from /config/www/community/list-card/list-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving bignumber-card/bignumber-card.js from /config/www/community/bignumber-card/bignumber-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-slider-entity-row/slider-entity-row.js from /config/www/community/lovelace-slider-entity-row/slider-entity-row.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving vertical-stack-in-card/vertical-stack-in-card.js from /config/www/community/vertical-stack-in-card/vertical-stack-in-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving bar-card/bar-card.js from /config/www/community/bar-card/bar-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving custom-header/custom-header.js from /config/www/community/custom-header/custom-header.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving button-card/button-card.js from /config/www/community/button-card/button-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving mini-graph-card/mini-graph-card.js from /config/www/community/mini-graph-card/mini-graph-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving mini-media-player/mini-media-player.js from /config/www/community/mini-media-player/mini-media-player.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving canvas-gauge-card/canvas-gauge-card.js from /config/www/community/canvas-gauge-card/canvas-gauge-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-hass-aarlo/hass-aarlo.js from /config/www/community/lovelace-hass-aarlo/hass-aarlo.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-auto-entities/auto-entities.js from /config/www/community/lovelace-auto-entities/auto-entities.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-card-mod/card-mod.js from /config/www/community/lovelace-card-mod/card-mod.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving surveillance-card/surveillance-card.js from /config/www/community/surveillance-card/surveillance-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving swipe-card/swipe-card.js from /config/www/community/swipe-card/swipe-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-template-entity-row/template-entity-row.js from /config/www/community/lovelace-template-entity-row/template-entity-row.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving banner-card/banner-card.js from /config/www/community/banner-card/banner-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-state-switch/state-switch.js from /config/www/community/lovelace-state-switch/state-switch.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-html-card/html-card.js from /config/www/community/lovelace-html-card/html-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving plan-coordinates/plan-coordinates.js from /config/www/community/plan-coordinates/plan-coordinates.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-fold-entity-row/fold-entity-row.js from /config/www/community/lovelace-fold-entity-row/fold-entity-row.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-card-tools/card-tools.js from /config/www/community/lovelace-card-tools/card-tools.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving lovelace-layout-card/layout-card.js from /config/www/community/lovelace-layout-card/layout-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving our-groceries-card/our-groceries-card.js from /config/www/community/our-groceries-card/our-groceries-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving flex-table-card/flex-table-card.js from /config/www/community/flex-table-card/flex-table-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving zha-network-card/zha-network-card.js from /config/www/community/zha-network-card/zha-network-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving upcoming-media-card/upcoming-media-card.js from /config/www/community/upcoming-media-card/upcoming-media-card.js.gz
2020-06-10 14:02:26 DEBUG (MainThread) [hacs] Serving entity-attributes-card/entity-attributes-card.js from /config/www/community/entity-attributes-card/entity-attributes-card.js.gz
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.ljmerza.calendar-card] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.bar-card] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-auto-entities] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.ljmerza.calendar-card] Running checks against 3.108.7
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.integration.twrecked.hass-aarlo] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.nervetattoo.banner-card] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.bignumber-card] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.entity-attributes-card] Getting repository information
2020-06-10 14:02:29 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.plan-coordinates] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-card-mod] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.bramkragten.swipe-card] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.bar-card] Running checks against 3.1.6
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-auto-entities] Running checks against 16
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.PiotrMachowski.lovelace-html-card] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.integration.twrecked.hass-aarlo] Running checks against v0.6.19.1
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.bignumber-card] Running checks against 0.0.2
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.nervetattoo.banner-card] Running checks against 0.10.0
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.integration.thomasloven.hass-browser_mod] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.plan-coordinates] Running checks against 0.1.1
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.bramkragten.swipe-card] Running checks against v3.2.1
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-card-mod] Running checks against 14
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.PiotrMachowski.lovelace-html-card] Running checks against v1.0.0
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.twrecked.lovelace-hass-aarlo] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.dmulcahey.zha-network-card] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.integration.ljmerza.ha-our-groceries] Getting repository information
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.custom-cards.entity-attributes-card] Running checks against 0.1.2
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.twrecked.lovelace-hass-aarlo] Running checks against v0.1.1
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.plugin.dmulcahey.zha-network-card] Running checks against master
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.template] 'states' is undefined
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.integration.thomasloven.hass-browser_mod] Running checks against 27
2020-06-10 14:02:30 DEBUG (MainThread) [hacs.repository.integration.ljmerza.ha-our-groceries] Running checks against 1.3.5
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.ljmerza.our-groceries-card] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.integration.zachowj.hass-node-red] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.maykar.custom-header] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-template-entity-row] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.integration.ntilley905.faastatus] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.gadgetchnnel.lovelace-card-preloader] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.integration.briis.weatherbit] Getting repository information
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.ljmerza.our-groceries-card] Running checks against 1.4.0
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.integration.zachowj.hass-node-red] Running checks against v0.4.4
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.maykar.custom-header] Running checks against 1.5.7
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.integration.briis.weatherbit] Running checks against v0.20
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.gadgetchnnel.lovelace-card-preloader] Running checks against 0.0.4
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.integration.ntilley905.faastatus] Running checks against v0.3.0
2020-06-10 14:02:35 DEBUG (MainThread) [hacs.repository.plugin.thomasloven.lovelace-template-entity-row] Running checks against 1.1.0
2020-06-10 14:02:36 DEBUG (MainThread) [hacs.template] 'states' is undefined
2020-06-10 14:02:41 DEBUG (MainThread) [queueman.manager] Queue execution finished for 37 tasks finished in 17.43 seconds
2020-06-10 14:02:41 DEBUG (MainThread) [hacs.data] Saving data
2020-06-10 14:12:23 DEBUG (MainThread) [hacs] Nothing in the queue
2020-06-10 14:17:23 DEBUG (MainThread) [hacs] Serving lovelace-card-preloader/lovelace-card-preloader.js from /config/www/community/lovelace-card-preloader/lovelace-card-preloader.js.gz
2020-06-10 14:17:23 DEBUG (MainThread) [hacs] Serving calendar-card/calendar-card.js from /config/www/community/calendar-card/calendar-card.js.gz
2020-06-10 14:17:23 DEBUG (MainThread) [hacs] Serving list-card/list-card.js from /config/www/community/list-card/list-card.js.gz
2020-06-10 14:17:23 DEBUG (MainThread) [hacs] Serving lovelace-slider-entity-row/slider-entity-row.js from /config/www/community/lovelace-slider-entity-row/slider-entity-row.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving bignumber-card/bignumber-card.js from /config/www/community/bignumber-card/bignumber-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving vertical-stack-in-card/vertical-stack-in-card.js from /config/www/community/vertical-stack-in-card/vertical-stack-in-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-card-tools/card-tools.js from /config/www/community/lovelace-card-tools/card-tools.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-layout-card/layout-card.js from /config/www/community/lovelace-layout-card/layout-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving our-groceries-card/our-groceries-card.js from /config/www/community/our-groceries-card/our-groceries-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving zha-network-card/zha-network-card.js from /config/www/community/zha-network-card/zha-network-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving flex-table-card/flex-table-card.js from /config/www/community/flex-table-card/flex-table-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving upcoming-media-card/upcoming-media-card.js from /config/www/community/upcoming-media-card/upcoming-media-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving entity-attributes-card/entity-attributes-card.js from /config/www/community/entity-attributes-card/entity-attributes-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving bar-card/bar-card.js from /config/www/community/bar-card/bar-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving custom-header/custom-header.js from /config/www/community/custom-header/custom-header.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving button-card/button-card.js from /config/www/community/button-card/button-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving mini-graph-card/mini-graph-card.js from /config/www/community/mini-graph-card/mini-graph-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving mini-media-player/mini-media-player.js from /config/www/community/mini-media-player/mini-media-player.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving canvas-gauge-card/canvas-gauge-card.js from /config/www/community/canvas-gauge-card/canvas-gauge-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-hass-aarlo/hass-aarlo.js from /config/www/community/lovelace-hass-aarlo/hass-aarlo.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-auto-entities/auto-entities.js from /config/www/community/lovelace-auto-entities/auto-entities.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-card-mod/card-mod.js from /config/www/community/lovelace-card-mod/card-mod.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving surveillance-card/surveillance-card.js from /config/www/community/surveillance-card/surveillance-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving swipe-card/swipe-card.js from /config/www/community/swipe-card/swipe-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-template-entity-row/template-entity-row.js from /config/www/community/lovelace-template-entity-row/template-entity-row.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving banner-card/banner-card.js from /config/www/community/banner-card/banner-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-state-switch/state-switch.js from /config/www/community/lovelace-state-switch/state-switch.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-html-card/html-card.js from /config/www/community/lovelace-html-card/html-card.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving plan-coordinates/plan-coordinates.js from /config/www/community/plan-coordinates/plan-coordinates.js.gz
2020-06-10 14:17:24 DEBUG (MainThread) [hacs] Serving lovelace-fold-entity-row/fold-entity-row.js from /config/www/community/lovelace-fold-entity-row/fold-entity-row.js.gz
2020-06-10 14:22:24 DEBUG (MainThread) [hacs] Nothing in the queue
```

</details>

**Comment by KTibow** [comment]
Huh, getting this with card-tools, pressed "reinstall" multiple times.

**Comment by lancelon** [comment]
@ludeeus - Firstly, THANKS so much for HACS. Great tool. Secondly, I wonder if we can improve the user experience? I recently upgraded thinking I'd done everything right (changed the folder URL from community_plugin to hacsfiles) and no...it tells me I've got 11 integrations not properly loading in lovelace... why?!?! I can't click reinstall and nothing I've tried to remedy the problem is working. Such a mess!

**Comment by jpljonkman** [comment]
> EDIT: I've just deleted the 'restriction-card' to try to reinstall it - only to find I can't find it anymore

I have the same problem. After removing a card, because reinstalling didn't work, the card was gone...

**Comment by JJFourie** [comment]
After installing a custom card using HACS, I edited the Resource created by the HACS install in `Configuration -> Lovelace Dashboards -> Resources`. I appended a version behind the resource path as shown below. Adding a version caused HACS to flag the Lovelace element as not loaded properly. Removing the version in the Resource definition also fixed the "problem". 

It seems the HACS logic uses the exact string when doing the validation, and does not ignore the version like it should (?) to be in line with the Lovelace logic. Better yet, add the version during install/upgrade, so that we don't need to delete the browser cache to force the browser to use the new version.

```
/hacsfiles/mini-graph-card/mini-graph-card-bundle.js?v=0.10.0
```
See [this explanation](https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins#3-import-the-plugin-in-your-lovelace-config#user-content-updating-a-plugin) for the reason to specify a version.
